src/test/java/com/chessmates/lichess/data/LichessDataServiceImplTest.groovy

```diff
@@ -345,7 +348,7 @@ class LichessDataServiceImplTest extends Specification {
         service.updateGames(players)
 
         then:
-        56 * gameRepository.save(_)
+        1 * gameRepository.saveAll(_)
```

Comment:
To keep the same logic here we'd want to be checking the list passed to saveAll has a length of 56.